icubam/db/sqlite.py

```diff
@@ -12,52 +14,65 @@ class SQLiteDB:
   """Wraps SQLite DB for bed counts."""
 
   def __init__(self, db_path: str):
-    """Given a token file and a sheet id, loads the sheet to be queried."""
-    self._db_path = db_path
-
-    if os.path.exists(db_path):
-      self._conn = sqlite3.connect(db_path)
-    else:
-      self._conn = sqlite3.connect(db_path)
-      self._create_table()
-
-  def _create_table(self):
-    self._conn.execute(
-      """CREATE TABLE icus
-                          (icu_id INTEGER NOT NULL PRIMARY KEY,
-                           icu_name TEXT UNIQUE, dept TEXT, city TEXT, lat REAL,
-                           long REAL, telephone TEXT )"""
+    """Initializes the database."""
+    self._metadata = MetaData()
+    self._icus = Table(
+        'icus',
+        self._metadata,
+        Column('icu_id', Integer, primary_key=True),
```

Comment:
Can we enforce relational constraints on this column: https://www.sqlite.org/foreignkeys.html?  It would avoid explicit checks in the code and also avoid serious corruption issues down the road.

Reply:
We already have one in users table, also added to bed_updates.